sdk/ai/azure-ai-inference/pom.xml

```diff
@@ -85,5 +88,43 @@
       <version>1.0.0</version> <!-- {x-version-update;com.knuddels:jtokkit;external_dependency} -->
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-monitor-opentelemetry-exporter</artifactId>
```

Comment:
is it used?

Reply:
yes, required for samples to work, it seemed to use in SPI way.

Reply:
Actually, samples need just `io.opentelemetry:opentelemetry-sdk-extension-autoconfigure`(a transitive dependency of `azure-monitor-opentelemetry-exporter`), updated to use that directly.